androidApp/src/main/java/com/mbta/tid/mbta_app/android/nearbyTransit/NearbyTransitView.kt

```diff
@@ -56,7 +58,8 @@ fun NearbyTransitView(
     val (pinnedRoutes, togglePinnedRoute) = managePinnedRoutes()
 
     val nearbyWithRealtimeInfo =
-        remember(
+        rememberOnDispatcher(
```

Comment:
question: Similar to above, I'm concerned about how we'll approach using this consistently, especially where we'll want rememberSaveable as well. Could the use of Dispatchers.Default be pushed to a lower level?